doc/leak_tracking/CONCEPTS.md

```diff
@@ -5,9 +5,22 @@ The text below is under construction.
 # Understand Memory Leak Tracking Concepts
 
 This page describes leak tracking concepts.
-See other information on memory leaks [here](../README.md).
+Read more about leak tracking in [overview](OVERVIEW.md).
 
-Before reading about leak tracking, understand [Dart memory concepts](https://docs.flutter.dev/development/tools/devtools/memory#basic-memory-concepts).
+Before reading about leak tracking concepts, understand [Dart memory concepts](https://docs.flutter.dev/development/tools/devtools/memory#basic-memory-concepts).
+
+## Object types
+
+### Instrumented disposables
+
+Instrumented disposable is a disposable class, instrumented to be tracked by tools
+like leak tracker, by dispatching its creation and disposal to the class
+FlutterMemoryAllocations (listened by the class LeakTracking)
+or directly to the class LeakTracking.
+
+### Package disposables
+
+All disposable objects defined in a package.
```

Comment:
can you define what it means to be a "disposable object"

Reply:
Disposables defined here: https://github.com/flutter/website/pull/10039
I will link them as soon as it is public.